src/Widgets/MprisGui.vala

```diff
@@ -164,52 +164,72 @@ public class Sound.Widgets.ClientWidget : Gtk.Box {
     construct {
         load_remote_art_cancel = new Cancellable ();
 
-        player_revealer = new Gtk.Revealer ();
-        player_revealer.reveal_child = true;
-        var player_box = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 0);
-
         background = new Gtk.Image ();
+
         mask = new Gtk.Image.from_resource ("/io/elementary/wingpanel/sound/image-mask.svg");
         mask.no_show_all = true;
         mask.pixel_size = 48;
+
         var overlay = new Gtk.Overlay ();
-        overlay.add (background);
-        overlay.add_overlay (mask);
+        overlay.can_focus = true;
         overlay.margin_start = 4;
         overlay.margin_end = 4;
         overlay.margin_bottom = 2;
-        overlay.can_focus = true;
-        var background_box = new Gtk.EventBox ();
-        background_box.add_events (Gdk.EventMask.BUTTON_PRESS_MASK);
-        background_box.button_press_event.connect (raise_player);
-        background_box.add (overlay);
-        player_box.pack_start (background_box, false, false, 0);
+        overlay.add (background);
+        overlay.add_overlay (mask);
 
-        var titles_events = new Gtk.EventBox ();
-        var titles = new Gtk.Box (Gtk.Orientation.VERTICAL, 0);
-        titles.set_valign (Gtk.Align.CENTER);
         title_label = new MaxWidthLabel (MAX_WIDTH_TITLE);
-        title_label.set_use_markup (true);
-        title_label.set_line_wrap (true);
-        title_label.set_line_wrap_mode (Pango.WrapMode.WORD);
-        title_label.set_ellipsize (Pango.EllipsizeMode.END);
+        title_label.use_markup = true;
+        title_label.ellipsize = Pango.EllipsizeMode.END;
         title_label.halign = Gtk.Align.START;
-        titles.pack_start (title_label, false, false, 0);
+        title_label.valign = Gtk.Align.END;
+
         artist_label = new MaxWidthLabel (MAX_WIDTH_TITLE);
-        artist_label.set_line_wrap (true);
-        artist_label.set_line_wrap_mode (Pango.WrapMode.WORD);
-        artist_label.set_ellipsize (Pango.EllipsizeMode.END);
+        artist_label.use_markup = true;
```

Comment:
It looks like markup is never used for artists, so I don't think we need to support markup here.

Reply:
I'm more inclined to leave it since it's possible it fixed some weird issue with some especially weird artist name